addons/l10n_ma/data/l10n_ma_chart_data.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="ISO-8859-15"?>
 <odoo>
         <!--     
-        Plan comptable g�n�ral pour le Maroc.
+        Plan comptable g�n�ral pour le Maroc.
```

Comment:
@caburj I think your ide did something wrong here, probably due to the encoding of the file in your IDE

Reply:
oh my. I'll check this out.